### DISCUSSION TITLE
Privacy policy?

### DISCUSSION THREAD

**Discussion by reissgo** [Glench/ExtPay#164] [General]
I've now read in two different places that ExtensionPay's privacy policy is at the bottom of the homepage: extensionpay.com... but I can't find it. Has it been removed?

**Comment by burkybang** [comment]
It's under the FAQ -> "What about user privacy?", but it would actually be great to be its own page.

**Comment by ba32107** [comment]
+1, it would be great to have a dedicated privacy policy page that we can link to.
